src/pages/patientView/SampleManager.tsx

```diff
@@ -269,6 +269,16 @@ class SampleManager {
         return this.samples.map((sample: ClinicalDataBySampleId) => sample.id);
     }
 
+    getSampleIdsIndex() {
+        let indexList: { [s: string]: number } = {};
```

Comment:
this is not really a "get".    i would call it sampleIdToIndexMap and make it computed

Reply:
fixed